CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ You can find out backwards-compatibility policy [here](https://github.com/hynek/
 ### Backwards-incompatible changes:
 
 - Support for Python 2.7, 3.5, and 3.6 has been dropped.
+- Added `data_as_text` and `data_as_bytes`
```

Comment:
this is not an backwards-incompatible change (it's `Added`) :)

also please add the PR link now that this is open.